stories/atoms/components/listItem/ListItem.stories.mdx

```diff
@@ -92,14 +112,28 @@ The `ListItem` allows to display an element of a [List](/docs/atoms-list--overvi
 ### Title
 
 The property `title` allows you to display a text which is the main title of the item.
+It can be either a simple string or a node element.
 
 <Canvas>
-  <Story name="Title">{() => <ListItem title="The title of the item." />}</Story>
+  <Story name="Title">
+    {() => (
+      <List>
+        <ListItem title="The title as a string" />
+        <ListItem
+          title={
+            <Typography color="inverted-text" fontWeight="bold">
```

Comment:
Here because just above you said "The title as a string", I would have written "The title as a node element" in order to fit more with the description.